tests/language-feature-scripts/lazy-loading-ghost-allows-protected-property-reflection-write.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Verifies that generated access interceptors allow protected property write via Reflection
+--FILE--
+<?php
+
+require_once __DIR__ . '/init.php';
+
+class Kitchen
+{
+    protected $sweets;
+}
+
+$factory = new \ProxyManager\Factory\AccessInterceptorValueHolderFactory($configuration);
```

Comment:
Thanks!

Marco Pivetta

http://twitter.com/Ocramius

http://ocramius.github.com/

On 1 July 2016 at 16:48, Ben Davies <notifications@github.com> wrote:

> In
> tests/language-feature-scripts/lazy-loading-ghost-allows-protected-property-reflection-write.phpt
> <https://github.com/Ocramius/ProxyManager/pull/299#discussion_r69310560>:
>
> > @@ -0,0 +1,26 @@
> > +--TEST--
> > +Verifies that generated access interceptors allow protected property write via Reflection
> > +--FILE--
> > +<?php
> > +
> > +require_once __DIR__ . '/init.php';
> > +
> > +class Kitchen
> > +{
> > +    protected $sweets;
> > +}
> > +
> > +$factory = new \ProxyManager\Factory\AccessInterceptorValueHolderFactory($configuration);
>
> renamed :)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Ocramius/ProxyManager/pull/299/files/615d2a758598e6e8f82620012f6576439b584e7a#r69310560>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAJakM7XfzB0E-pcMz_C9lFNPnsKLZYlks5qRSi0gaJpZM4HqtIb>
> .
>